requirements-testing.txt

```diff
@@ -5,6 +5,10 @@ pytest == 7.*
 pytest-cov == 5.*
 pytest-timeout == 2.*
 pytest-xdist == 3.*
+pytest-qt == 4.*
+pytest-xvfb == 3.*
+PySide6-essentials == 6.6.2; python_version == '3.8'
+PySide6-essentials == 6.7.1; python_version > '3.8'
```

Comment:
Should we not have a range of supported versions for testing as well? We can probably just do `>= 6.6.*`